src/scores.ts

```diff
@@ -100,7 +100,7 @@ const pendingRequests = {};
 
 export async function updateProposalAndVotes(proposalId: string, force = false) {
   const proposal = await getProposal(proposalId);
-  if (!proposal) return false;
+  if (!proposal || proposal.state === 'pending') return false;
```

Comment:
But `getProposal` doesn't return `state` property no?

Reply:
It's used 3 lines below, at line 106, so I believe it's set

Reply:
Oh its set, i was looking at different `getProposal`